main.go

```diff
@@ -67,8 +68,46 @@ func usage() {
 }
 
 func main() {
+	log.SetFlags(0)
```

Comment:
It was more of a drive-by change because time/date information was being logged as I played around...considered is sufficiently innocuous to include in this PR. But happy to split into a separate PR if you'd prefer